src/tools/rbd/action/ImageMeta.cc

```diff
@@ -94,6 +94,24 @@ static int do_metadata_set(librbd::Image& image, const char *key,
   return r;
 }
 
+static int do_matadata_keycheck(librbd::Image& image, const char *key){
+    std::map<std::string, bufferlist> pairs;
+    int r;
+    r = image.metadata_list("", 0, &pairs);
```

Comment:
Once moved, use the internally available ```librbd::cls_client::metadata_get``` function to retrieve just the specific key